Sprint-3/implement/get-card-value.js

```diff
@@ -29,3 +29,23 @@
 // Given a card with an invalid rank (neither a number nor a recognized face card),
 // When the function is called with such a card,
 // Then it should throw an error indicating "Invalid card rank."
+
+function getCardValue(card) {
+  let rank = card.slice(0, -1);
+  const parseIntRank = parseInt(rank);
```

Comment:
It's useful to differentiate variable names when you have two things which have the same (but for some reason different, e.g. a different type type) data in them.

I probably _wouldn't_ use `parseIntRank` as a variable name - `parse` is a verb, and verbs suggest _doing_ (i.e. function names).

What do you think of each of these variable name pairs?

`rankString` and `rankNumber`

`rankAsString` and `rankAsNumber`

`stringRank` and `numericRank`

`rankString` and `rank`